Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -155,36 +156,6 @@ static int CompareGameListItems(const GameListItem* iso1, const GameListItem* is
   return 0;
 }
 
-static std::vector<std::string> GetFileSearchExtensions()
```

Comment:
The idea with new cache design is that the cache holds data for all files in the search path. This means that only the cache needs to be consulted for operations that are expected to be lightweight, such as changing which columns are displayed.